app/code/Magento/GiftMessageGraphQl/Model/Resolver/GiftMessageItem.php

```diff
@@ -0,0 +1,45 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\GiftMessageGraphQl\Model\Resolver;
+
+use Magento\Framework\GraphQl\Config\Element\Field;
+use Magento\Framework\GraphQl\Query\Resolver\ContextInterface;
+use Magento\Framework\GraphQl\Query\Resolver\Value;
+use Magento\Framework\GraphQl\Query\ResolverInterface;
+use Magento\Framework\GraphQl\Schema\Type\ResolveInfo;
+
+/**
+ * Class GiftMessageItem
+ */
+class GiftMessageItem implements ResolverInterface
```

Comment:
I'm not sure we should extend this PR with more functionality. Let's leave the gift messages for different cart item types in this PR and cover all other functionality (mutations, gift wrapping) in the following PRs

Reply:
I have removed it from this PR, will do it in other PR